src/vue/composition.h.ts

```diff
@@ -1,8 +1,8 @@
 import {Domain, Event, Store} from 'effector'
 
 type Gate<Props> = {
-  open: Event<Props>;
-  close: Event<Props>;
+  open: Event<Props | void>;
```

Comment:
🤔 I do not get it, why `open` can be called without props?

Reply:
@igorkamyshev why not? Sometimes I can use gate without passing props

Reply:
As I understand particular Gate should always be called with props or without props.

So I suggest to use Props type here and pass Props = void, if you are not going to use props.

Reply:
Thanks! Fixed that